godot-headers/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright (c) 2017-2022 Godot Engine contributors.
+
```

Comment:
Since the license is identical to the `godot-cpp` license, probably can be removed as well.